.github/workflows/test.yml

```
@@ -0,0 +1,43 @@
+name: Go
+
+on: [push]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: [1.16, 1.17, 1.18]
```

```suggestion
        go-version: [1.18, 1.19]
```
might want to address these